src/config/musl.md

```diff
@@ -0,0 +1,67 @@
+# musl
+
+[musl](https://musl.libc.org/) is a libc implementation which strives to be
+lightweight, fast, simple, and correct.
+
+Void officially supports musl by using it in its codebase for all target
+platforms (although binary packages are not available for i686). Additionally,
+all compatible packages in our official repositories are available with
+musl-linked binaries in addition to their glibc counterparts.
+
+Currently, there are nonfree and debug subrepositories for musl, but no multilib
+subrepo.
+
+## Possible problems
+
+In some cases, software compatibility may be worse with musl.
+
+Proprietary software rarely supports non-glibc libc implementations, although
+sometimes these applications are available as [flatpaks](https://flatpak.org/),
+which provide their own libc in the image.
+
+### glibc chroot
+
+Software requiring glibc can be run in a glibc chroot.
+
+Create a directory that will contain the chroot, and install a base system in it
+via the `base-voidstrap` package. If network access is required, copy
+`/etc/resolv.conf` and `/etc/hosts` into the chroot.
```

Comment:
I'm not sure `/etc/hosts` should be recommended here. There are a lot of chroot applications where this would be unnecessary and/or undesireable.

Reply:
i added this following @q66's comment [here](https://github.com/void-linux/void-docs/pull/163#discussion_r395958839), but i'm certainly happy to remove it if @q66 is fine with that. :-)

Reply:
Oh, sorry i didn't catch that. /etc/hosts isn't strictly required for network access. I don't feel super strongly about this. I guess I would be fine if `/etc/hosts` was qualified.

Reply:
Updated.